types/Jest.js

```diff
@@ -43,7 +43,7 @@ export type Jest = {|
   runTimersToTime(msToRun: number): void,
   setMock(moduleName: string, moduleExports: any): Jest,
   setTimeout(timeout: number): Jest,
-  spyOn(object: Object, methodName: string): JestMockFn,
+  spyOn(object: Object, methodName: string, accessType?: string): JestMockFn,
```

Comment:
Can you document the new argument? https://github.com/facebook/jest/blob/9cbc26bab656fa9e9952242b2d733a86c6a61d43/docs/JestObjectAPI.md#jestspyonobject-methodname

Reply:
i've added some basic documentation for the new parameters but i think that it should be finalized. (put the right version and eventually merge the two definitions, i've left them separate because of the different jest version since they are available)